docs/Classes.html

```diff
@@ -21,7 +21,7 @@
     <header class="header">
       <p class="header-col header-col--primary">
         <a class="header-link" href="index.html">
-          Frames 3.5.2 Docs
+          Frames 3.5.3 Docs
```

Comment:
Again, these doc changes will be overridden with what we already have in checkout repo.